test/e2e/mmi/pageObjects/mmi-accountMenu-page.ts

```diff
@@ -144,7 +145,15 @@ export class MMIAccountMenuPage {
       accountNames.push(accountName);
     }
 
-    await this.page.getByRole('button', { name: /close/iu }).click();
+    await this.page.getByRole('button', { name: /close/iu }).first().click();
     return accountNames;
   }
+
+  async closeBanner() {
+    await this.page
+      .getByRole('dialog')
+      .filter({ hasText: 'Select an account' })
+      .locator('.mm-banner-base__close-button')
```

Comment:
Nit: Typically we try to avoid classname locators because they can lead to churn (classnames change) and because they can lead to mistakes where a new instance of the class is introduced, leading to the test clicking the wrong thing. I see that there is no obvious better choice in this case though.

Maybe it would be better to introduce a `data-testid` for this banner, and select that here instead, followed by a `getByRole` to get the close button. But this does not seem that important, and I am eager to see this test passing again :smile: